File: demojava/src/main/java/com/teskalabs/seacat/demojava/SeaCatDemoApp.java
package com.teskalabs.seacat.demojava;

import android.app.Application;

import com.teskalabs.seacat.SeaCat;

public class SeaCatDemoApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        SeaCat.init(this, "https://pki.seacat.io/seacat-demo/seacat");
    }
}

File: demo/src/main/java/com/teskalabs/seacat/demo/SeaCatDemoApp.kt
package com.teskalabs.seacat.demo

import android.app.Application
import com.teskalabs.seacat.SeaCat

class SeaCatDemoApp : Application() {
    override fun onCreate() {
        super.onCreate()
        SeaCat.init(this, "https://pki.seacat.io/seacat-demo/seacat")
    }
}
